src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
@mediumTaj  Yes, but for this your method should return a string and not ClassifyTopLevelMultiple, or return the Request, but this implies some issues, such as giving the developer responsibility to "translate" the return correctly.